multi-casskop/deploy/operator.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: multi-casskop
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: multi-casskop
+  template:
+    metadata:
+      labels:
+        name: multi-casskop
+    spec:
+      serviceAccountName: multi-casskop
+      containers:
+        - name: multi-casskop
+          # Replace this with the built image name
+          image: REPLACE_IMAGE
```

Comment:
Let's just put the comment in here and remove it on line 18
`image: "Replace this with the built image name"`

Reply:
wonder if I don't have a script that uses the REPLACE_IMAGE, I prefer keep it for now